minvws/nl-kat-coordination
# Fix boefje schema on Boefje Setup page

### Changes
Because of the change of the "schedule" variable to "boefje_variable", the Setup Boefje page collected the the wrong data.

### Issue link
Closes #3569 

### Demo
Boefje variant of Nmap TCP now shows the correct schema:
![afbeelding](https://github.com/user-attachments/assets/1c9a257e-b8f1-4a5c-8fa2-be351290ed5d)


### QA notes
- Go to Nmap TCP detail page
- Click on "Add variant"
- Check if the scheduled data is correct

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
